NEWS

```diff
@@ -1,5 +1,27 @@
 systemd System and Service Manager
 
+CHANGES WITH 249:
+
+        * The runtime dependency on libbpf, has been changed to be based on
```

Comment:
The runtime dependency on libbpf, has been changed to be based on …
↓
The runtime dependency on libbpf has been changed to use …

Hmm, I think it'd be better to keep it short, i.e. more like the description for `libfido2` below. Maybe skip all the explanations and say "... similarly to libqrencode, libpcre2, and others; see below for more details."

Reply:
actually I'll drop this commit, given libbpf is new to 249, no need to add a notice - it was never a linked dependency